include/sys/spa.h

```diff
@@ -555,13 +675,14 @@ _NOTE(CONSTCOND) } while (0)
 		    DVA_GET_ASIZE(&bp->blk_dva[1]) / 2)			\
 			copies--;					\
 		len += func(buf + len, size - len,			\
-		    "[L%llu %s] %s %s %s %s %s %s%c"			\
+		    "[L%llu %s] %s %s %s %s %s %s %s%c"			\
```

Comment:
Sure. Should I bump up `BP_SPRINTF_LEN` along with this? or should that stay at 320?